testing/workflows/components/workflows.libsonnet

```diff
@@ -166,17 +194,30 @@
                 ],
               },
             },  // checkout
-            $.parts(namespace, name).e2e(prow_env, bucket).buildTemplate("test-deploy", [
+            $.parts(namespace, name).e2e(prow_env, bucket).buildTemplate("setup", [
               "python",
               "-m",
               "testing.test_deploy",
```

Comment:
do you need to add "setup" for the subparser?

Reply:
Its the last argument. The order of the arguments matters because the first set of parameters are not parameters of the subparser setup so the need to be specified before the subparser command.